spec/features/saved_searches_spec.rb

```diff
@@ -10,6 +10,18 @@
     click_link 'Saved Searches'
     expect(page).to have_content 'You have no saved searches'
   end
+  
+  it 'can be saved and forgotten from a search result' do
+    visit catalog_index_path(q: 'book')
+    within '.search-widgets' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.